dup_struct_name_test.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 import (
-	dup "dubbo-go-hessian2-dup-struct-name-test"
```

Comment:
why not use the module name in hessian2 project？

Reply:
Old version